src/app/content/components/Topbar/styled.tsx

```diff
@@ -0,0 +1,315 @@
+      display: none;
+    `)}
+  `;
+
+// tslint:disable-next-line:variable-name
+export const CloseIcon = styled((props) => <Times {...props} aria-hidden='true' focusable='false' />)`
+    color: ${toolbarIconColor.base};
+    height: 2.2rem;
+  `;
+
+// tslint:disable-next-line:variable-name
+export const CloseButtonNew = styled.button`
+    ${toolbarIconStyles}
+    cursor: pointer;
+    border: none;
+    padding: 0;
+    margin-right: 1.6rem;
+    background: transparent;
+    overflow: visible;
+    height: 2.2rem;
+    width: 2.2rem;
+  `;
+
+// tslint:disable-next-line:variable-name
+export const SearchInputWrapper = styled.form`
+    display: flex;
+    align-items: center;
+    margin-right: 2rem;
+    position: relative;
+    color: ${toolbarIconColor.base};
+    border: solid 0.1rem;
+    border-radius: 0.2rem;
+    width: 38rem;
+
+    &:focus-within {
+      border: solid 0.1rem ${theme.color.secondary.lightBlue.base};
+      box-shadow: 0 0 4px 0 rgba(13, 192, 220, 0.5);
+    }
+
+    &.focus-within {
+      border: solid 0.1rem ${theme.color.secondary.lightBlue.base};
+      box-shadow: 0 0 4px 0 rgba(13, 192, 220, 0.5);
+    }
+
+    ${theme.breakpoints.mobileMedium(css`
+      margin-right: 0;
+      height: 100%;
+      overflow: hidden;
+      width: 100%;
+      ${(props: { active: boolean, colorSchema: BookWithOSWebData['theme'] }) => props.active && css`
+        background: ${props.colorSchema ? theme.color.primary[props.colorSchema].base : 'transparent'};
+
+        ${SearchButton} {
+          ${applySearchIconColor(props.colorSchema)};
+        }
+      `}
+    `)}
+  `;
+
+// tslint:disable-next-line:variable-name
+export const SearchInput = styled(({ desktop, mobile, ...props }) =>
+  <input {...props}
+    aria-label={useIntl().formatMessage({ id: 'i18n:toolbar:search:placeholder' })}
+    placeholder={useIntl().formatMessage({ id: 'i18n:toolbar:search:placeholder' })}
+  />)`
+      ${textStyle}
+      ${hideSearchChrome}
+      font-size: 1.6rem;
+      margin: 0 1rem 0 1rem;
+      height: ${toolbarSearchInputHeight}rem;
+      border: none;
+      outline: none;
+      width: 100%;
+      appearance: textfield;
+
+      ::placeholder {
+        color: ${theme.color.text.label};
+      }
+
+      ${(props) => props.desktop && theme.breakpoints.mobileMedium(css`
+        display: none;
+      `)}
+    `;
+
+// tslint:disable-next-line:variable-name
+export const SearchPrintWrapper = isVerticalNavOpenConnector(styled.div`
+  height: ${topbarDesktopHeight}rem;
+  max-width: ${contentWrapperMaxWidth}rem;
+  margin: 0 auto;
+  text-align: right;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  overflow: visible;
+  background-color: ${theme.color.neutral.base};
+  transition: padding-left ${sidebarTransitionTime}ms;
+  ${(props) => (props.isVerticalNavOpen === null || props.isVerticalNavOpen) && `
+    padding-left: ${sidebarDesktopWidth}rem;
+  `}
```

Comment:
there used to be helpers for managing the sidebar like `styleWhenSidebarOpen`, not only did this contain the `null` checking logic so that that was handled in one place, it also handled adding multiple media queries so that the null condition was handled properly at different screen sizes.

some of your comments indicated some trouble managing the permutations of sidebar state and null state, you might find it helpful to bring that pattern back